types/styled-jsx/server.d.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+declare function flushToHTML(): string;
+declare function flush(): React.ReactChildren;
```

Comment:
not familiar with TypeScript but the return type here I think is an `Array<React.Element>`